### DISCUSSION TITLE
Move Translations out of the Data Model main view into it's own section

### DISCUSSION THREAD

**Discussion by damien-mcmahon** [directus/directus#21742] [Feature Requests]
### Summary

When working in the Data Model section of the settings area in Directus, we've found that it's annoying to have to see all the translation models in the same section. Can we have a Translations section similar to the System Collections?
----
<img width="1245" alt="Screenshot 2024-03-07 at 13 34 05" src="https://github.com/directus/directus/assets/17355496/ee4d28fd-91d2-4b6d-ac7e-d8614fba14aa">

This is a small selection of the maybe 100+ translation models we have to scroll past when creating a new model and trying to group it with our actual models at the top of the list

### Basic Example

There's already a System Collections, please add a Translation Collections group
<img width="464" alt="Screenshot 2024-03-07 at 13 35 59" src="https://github.com/directus/directus/assets/17355496/344e6350-1975-49f8-a376-bbf3a5e0a2c3">

### Motivation

Makes working with large collections of translations easier

### Detailed Design

Follow the design of the Systems Collection and any new translations that are added live in the new section, perhaps with the ability to filter inside of the collapsible area

### Must Have:
- A section for Translations that's collapsible

### Nice To Have:
- Filterable search per section

### Drawbacks

Not many, it's a small UI change and would aid in the UX of the product

### Alternatives

Enable users to define custom groupings of the data models

### Adoption Strategy

It's part of the UI, not needed

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
